src/index.css

```diff
@@ -24,3 +34,50 @@
 main {
   flex: 1;
 }
+
+/*The CSS class "sr-only" is used to hide element visually from
+sighted users while still making them accessible to assistive
+technologies such as screen readers (SR), making website more 
+inclusive for all users. That's why the name of the class is 
+"sr-only", which means "for Screen Readers only". 
+*/
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  white-space: nowrap;
+  border: 0;
+}
+
+html {
+  font-size: 100%;
+}
+
+/* Responsive Design Tablet */
+
+@media screen and (max-width: 991px) {
+  html {
+    /*width < 900?*/
+    font-size: 80%;
+  }
+}
+
+/* Responsive Design Mobile */
+@media screen and (max-width: 600px) {
```

Comment:
Good job making your app responsive! I do strongly recommend using EM units in media query always!  Here is the good article about why it is the best choice - https://zellwk.com/blog/media-query-units/